src/types/index.ts

```diff
@@ -1,16 +1,31 @@
 export type Command = () => void
 export type AsyncCommand = () => Promise<void>
 
-export interface TokenDetails {
```

Comment:
@Velenir  commented
> You removed TokenDetails completely?

@anxolin 
> hehe, that's the controversial part ;)
>
>The object with the same shape is called now TokenDex, it's a token that is listed in our Exchange. Only difference >with Token is that the latter has optional tokenId, so it may or may not be listed in the exchange
>
>Then the other relevant token defintion is TokenErc20 that models a ERC20 (well, with mandatory decimals, we are taking a small liberty there)